Library/Formula/spandsp.rb

```diff
@@ -0,0 +1,34 @@
+class Spandsp < Formula
+  desc "DSP functions library for telephony"
+  homepage "http://www.soft-switch.org"
+  url "http://www.soft-switch.org/downloads/spandsp/spandsp-0.0.6.tar.gz"
+  sha256 "cc053ac67e8ac4bb992f258fd94f275a7872df959f6a87763965feabfdcc9465"
+
+  depends_on "libtiff"
+
+  def install
+    ENV.deparallelize
```

Comment:
There's no public bug tracker for spandsp. The spandsp mailing list looks like the only public forum.

http://lists.soft-switch.org/mailman/listinfo
http://lists.soft-switch.org/pipermail/spandsp/

The archives are not searchable; might be best to just fire off a new bug report message.